data/pileup/jec18ul/do_pileup.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
```

Comment:
Shouldn't this script be more general than only UL18?

Reply:
Yes, it probably can be generalized. That would require command-line parameters though, which are a bit of a pain to do in bash. Reimplementing the whole think in Python would be an alternative, but then it just feels like an unnecessary wrapper around `puWeightCalc.py`.

Reply:
Not sure what you mean with command line parameters. Can't you just give command line arguments to the script (`$0`, `$1`, etc.), which are then interpreted?

Reply:
Sure, but I mean something nicer like `--input-file` and the like. Looks like this can be achieved with `getopt`. Will look into it.